training/train_utils.py

```diff
@@ -217,10 +217,8 @@ def load_checkpoint(checkpoint, model, optimizer=None):
     """
     if not os.path.exists(checkpoint):
         raise FileNotFoundError("File doesn't exist {}".format(checkpoint))
-    if torch.cuda.is_available():
-        checkpoint = torch.load(checkpoint)
-    else:
-        checkpoint = torch.load(checkpoint, map_location='cpu')
```

Comment:
Is this just a refactor or a patch?

Reply:
Agree that this is a bit out of scope. I personally don't mind it since it would be overkill to make a dedicated pr, but the commit should at least mention it

Reply:
Yup. Fine by me.